overlord/snapstate/aliasesv2.go

```diff
@@ -104,11 +140,207 @@ func applyAliasesChange(st *state.State, snapName string, prevStatus AliasesStat
 			})
 		}
 	}
-	st.Unlock()
 	err := be.UpdateAliases(add, remove)
-	st.Lock()
 	if err != nil {
 		return err
 	}
 	return nil
 }
+
+// autoAliasesDeltaV2 compares the automatic aliases with the current snap
+// declaration for the installed snaps with the given names (or all if
+// names is empty) and returns touched and retired auto-aliases by
+// snap name.
+// TODO: temporary name
+func autoAliasesDeltaV2(st *state.State, names []string) (touched map[string][]string, retired map[string][]string, err error) {
```

Comment:
I s/touched/changed/  ,  I went for s/retired/dropped/  , the concept of retire(d) was already in the old code btw, and we need to name a task after it as well, usually we use "remove" more for final low-level removing of files/links from disk in that context (that's why we already also have clear and discard in task names). Anyway we can revisit this again in a later PR where the whole context is more clear.